### DISCUSSION TITLE
cloud do please make lora-script support on this code?

### DISCUSSION THREAD

**Discussion by Canyayan** [gutris1/segsmaker#37] [Ideas]
Thank you very much, your code is amazing! I can finally use my sagemaker now.
And I really want it can support lora-script, will you consider it ?

**Comment by gutris1** [comment]
what do you mean by lora-script?

**Reply by Canyayan** [reply]
Thank you for your reply!!  I want your code to combine the function of LoRA training. I just git this（https://github.com/Akegarasu/lora-scripts）after install webui with your code, but it doesn't work since I don't know how to access http://127.0.0.1:28000/ on sagemaker.

I mean your code is really helpful when I use webui for painting, how about add the lora training in your code, and make it an extra for us who like to traning and test on sagemaker?

**Reply by gutris1** [reply]
nope.
